Source/WebKitLegacy/mac/History/WebHistory.mm

```diff
@@ -291,8 +291,6 @@ - (WebHistoryItem *)visitedURL:(NSURL *)url withTitle:(NSString *)title
         // as seen in <rdar://problem/6570573>.
         BOOL itemWasInDateCaches = [self removeItemFromDateCaches:entry.get()];
         ASSERT_UNUSED(itemWasInDateCaches, itemWasInDateCaches);
-
-        [entry _visitedWithTitle:title];
```

Comment:
I think this looks risky. This method was setting the title sure, but it was also setting the last visited time. The last visited time is no longer getting set now, which seems like an unnecessary behavior change?

I suggest we name this to `_visited`

Reply:
```
        [entry _visitedWithTitle:title];
```

> I suggest we name this to `_visited`

OK, made that change